### PR TITLE
Additional interop helper methods

### PR DESCRIPTION
**Pull request by casselc**
Includes helpers for `->CookieHandler`, `->SSLParameters`, and `->Executor` which are incorporated into the `client-builder`.

Please answer the following questions and leave the below in as part of your PR.

- [x] I have read the [developer documentation](https://github.com/babashka/babashka/blob/master/doc/dev.md).

- [x] This PR corresponds to an #34 

- [x] This PR contains a [test](https://github.com/babashka/babashka/blob/master/doc/dev.md#tests) to prevent against future regressions

- [x] I have updated the [CHANGELOG.md](https://github.com/babashka/babashka/blob/master/CHANGELOG.md) file with a description of the addressed issue.

### CODE REVIEW

**Comment by casselc** [comment]
I think https://www.httpbin.org/ may be having issues. I've been seeing flakiness/occasional 504 gateway timeouts in local testing of `get-response-object-test`, and seeing similar behavior trying to access https://httpbin.org/redirect-to?url=https://www.httpbin.org in a browser.

File: src/babashka/http_client/internal.clj [hunk 1]
```diff
@@ -112,6 +112,42 @@
...
+        (java.net.CookieManager. store policy)))))
+
+(defn ->SSLParameters
+  [v]
+  (when v
+    (if (instance? javax.net.ssl.SSLParameters v)
+      v
+      (let [{:keys [ciphers protocols]} v
+            params (javax.net.ssl.SSLParameters.)]
+        (when (not-empty ciphers)
```
Comment (borkdude) [inline src/babashka/http_client/internal.clj:137]: I think `seq` is more appropriate here than `not-empty`

**Comment by borkdude** [comment]
Maybe we should replace those tests with something else (as a separate PR)

**Comment by borkdude** [comment]
Merged manually

**Pull request closed without merging**